tests/storage/datavolume.go

```diff
@@ -381,6 +383,56 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 					return vm.Status.PrintableStatus
 				}, 180*time.Second, 2*time.Second).Should(Equal(v1.VirtualMachineStatusStopped))
 			})
+
+			It("should accurately aggregate DataVolume conditions from many DVs", func() {
+				sc, exist := libstorage.GetRWOFileSystemStorageClass()
+				if !exist {
+					sc, exist = libstorage.GetRWXFileSystemStorageClass()
+					if !exist {
+						Skip("Skip test when FileSystem storage is not present")
```

Comment:
Please do not use skip but rather fail.

Reply:
I am sorry but in this case, the skip is wrong, and being consistent would be wrong as well. I acknowledge that we have not yet removed this problem but I can't accept new changes that introduce more of it.

Please use `RequiresRWXFilesystemStorage` in `tests/decorators` for the test and the skip can be safely removed.

Reply:
it's even simpler: there is no reason while this should work just with RequiresRWXFilesystemStorage and not with other kinds, we already have `decorators.StorageReq` on the top-level Context so we can directly drop the check.